spec/controllers/courses_controller_spec.rb

```diff
@@ -261,6 +261,38 @@
         end
       end
 
+      context 'when blank values are given for course school, title or both' do
+        let(:course_params) do
+          { school: ' ',
+            title: '  ',
+            term: 'Fall 2015',
+            start: '2015-01-05',
+            end: '2015-12-20',
+            language: 'arrr',
+            project: 'wikipirates' }
```

Comment:
You've copied the language and project from another invalid example, so this is not testing your change. These tests all pass even without your change to CourseCreationManager. Use 'en' and 'wikipedia' instead.

One way to improve these new tests as well as the old ones would be to test for the expected error message. For example, `expect(response.body).to have_content('Blank school/title for course')`.